requirements-sqlite.txt

```diff
@@ -0,0 +1,13 @@
+opster
```

Comment:
Should there be a core `requirements.txt` file, then such files for each backend?

Reply:
Actually after thinking about this, I'm not so sure because here's how I see deployment happening. We have a configs folder with the three pairs of files in it, `requirements-sqlite.txt`, `endpoint-sqlite.config` and `requirements-mongodb.txt`, `endpoint-mongodb.config` and `requirements-google-ndb.txt`, `endpoint-google-ndb.config` and we have instructions that the appropriate requirements file be copied to root, and the appropriate endpoint config file be copied to `endpoint/endpoint.cfg` and then the app is ready for deployment.